README.md

```diff
@@ -73,6 +73,46 @@ There are variable defaults set in both the `common.sh` and the `ocp_install_env
 scripts, which may be important to override for your particular environment. You can
 set override values in your `config_$USER.sh` script.
 
+### Configuration for 64-bit ARM 
```

Comment:
Does the ARM require to run dev-scripts on a ARM host?

Reply:
yeah, the host(s) must be ARM.

Reply:
Could you please add this requirement in the doc? There's no check preventing that, so it could be misleading if accidentally run on another arch. For the rest lgtm to me.